content/applications/marketing/events/overview/create.rst

```diff
@@ -0,0 +1,298 @@
+
+Log internal notes or add ticket instructions
+---------------------------------------------
+
+In the :guilabel:`Notes` tab, there is the option to add a :guilabel:`Note` and/or
+:guilabel:`Ticket instructions`.
+
+Under :guilabel:`Note`, internal notes can be left (such as: to-do lists, contact information,
+etc.) for event staff to reference. In the :guilabel:`Ticket instructions` field, helpful
+information for staff and attendees can be shared (like, directions to the venue, opening/closing
+times, etc.)
+
+.. note::
+   Type *"/"* in either text field (:guilabel:`Note` or :guilabel:`Ticket instructions`) to reveal
+   a sub-menu of :guilabel:`Structure` options. These options provide various formatting options to
+   ensure vital internal information is organzied for event staff to review.
+
+.. image:: create/events-notes-tab.png
+   :align: center
+   :alt: View of the Notes tab in Odoo Events.
+
+Event invites
+-------------
+
+To invite people to an event, click :guilabel:`Invite`.
+
+.. image:: create/events-invite-button.png
+   :align: center
+   :alt: View of an event form and the invite button in Odoo Events.
+
+This opens an invite template form, that can be emailed or sent out as an SMS. The message can be
+fully customized, and recipients can be added.
+
+Adding a :guilabel:`Subject` line for the invite message is required, but the
+:guilabel:`Preview Text` field is optional. The :guilabel:`Preview Text` is a catchy preview
+sentence meant to encourage recipients to open the email.
+
+.. note::
+   In most cases, the :guilabel:`Preview Text` is displayed next to the subject. Keep this field
+   empty to ensure the first characters of the email content appear, instead.
+
+Select invitees and configure recipient filters
+-----------------------------------------------
+
+Click the :guilabel:`Recipients` field, to reveal a drop-down menu of recipient options. These
+choices represent where Odoo will find the desired recipients' information.
+
+Once an option from this menu is selected (e.g. *Applicant*, *Contact*, *Event Registration*,
+*Lead/Opportunity*, etc.), Odoo can send it to all recipients who match that initial rule, or a
+rule(s) can be added (and configured) to narrow down the target recipients, by clicking
+:guilabel:`Add Filter`.
+
+.. image:: create/add-filter-button.png
+   :align: center
+   :alt: View of the add filter button beneath the recipients field in Odoo Events.
+
+Clicking :guilabel:`Add Filter` reveals three fields, formatted like an equation. To reveal the
+sub-menu options, click each field, and make the desired selections, until the preferred
+configuration has been acheived. The number of *Records* that match the rule(s) are indicated to the
+right of the :guilabel:`Recipients` field, in green.
+
+.. image:: create/recipient-filters.png
+   :align: center
+   :alt: View of the recipient filter fields used for event invite messages.
```

Comment:
Since I provided a more detailed sample of this kind of recipient filter this time around (for the suggested image above), I think I can safely omit this image, without confusing the reader.